docs/articles_en/openvino-workflow/running-inference/inference-devices-and-modes/npu-device.rst

```diff
@@ -13,27 +13,23 @@ Intel® Core™ Ultra generation of CPUs (formerly known as Meteor Lake). It ena
 you to offload certain neural network computation tasks from other devices,
 for more streamlined resource management.
 
-Note that the NPU plugin is currently available only with the Archive distribution of OpenVINO™
-and you need to :doc:`install a proper NPU driver <../../../get-started/configurations/configurations-intel-npu>`
-to use it successfully.
+NPU Plugin is now available through all relevant OpenVINO distribution channels.
```

Comment:
@mlyashko @ilya-lavrenov can you please confirm we can state this?

Reply:
currently - no, but it's WIP during OpenVINO 2024.1 publishing.
So, let's keep it in current form assuming we complete this work